Makefile

```diff
@@ -31,27 +22,17 @@ all:
 	@echo
 	@echo "Package requirements related targets"
 	@echo "requirements-dev:      Install development requirements"
-	@echo "requirements-plugins:  Install plugins requirements"
 	@echo
 	@echo "Platform independent distribution/installation related targets:"
-	@echo "source:       Create single source package with commit info, suitable for RPMs"
 	@echo "source-pypi:  Create source packages suitable for PyPI"
 	@echo "python_build: Installs the build package, needed for source-pypi and wheel"
 	@echo "install:      Install on local system"
 	@echo "uninstall:    Uninstall Avocado and also subprojects"
 	@echo "man:          Generate the avocado man page"
 	@echo
-	@echo "RPM related targets:"
-	@echo "srpm:  Generate a source RPM package (.srpm)"
-	@echo "rpm:   Generate binary RPMs"
-	@echo
-	@echo "Release related targets:"
-	@echo "source-release:  Create source package for the latest tagged release"
-	@echo "srpm-release:    Generate a source RPM package (.srpm) for the latest tagged release"
-	@echo "rpm-release:        Generate binary RPMs for the latest tagged release"
-	@echo
 
-include Makefile.include
+pip:
+	$(PYTHON) -m pip --version || $(PYTHON) -m ensurepip $(PYTHON_DEVELOP_ARGS) || $(PYTHON) -c "import os; import sys; import urllib; f = urllib.urlretrieve('https://bootstrap.pypa.io/get-pip.py')[0]; os.system('%s %s' % (sys.executable, f))"
```

Comment:
I understand that you are basically moving things from a different place here, but since this target has no help string, It would be much appreciated if we had a proper help message for this target.

Reply:
I'm not sure that would be a good idea. This is an auxiliary target and we don't want people using it directly. It's just a help for people who don't have pip in their systems.

Reply:
IMO all targets could have a help message. Specially this case, so a disclaimer saying "Use this target only if you don't have pip installed".